lbrynet/core/LBRYWallet.py

```diff
@@ -534,21 +575,38 @@ def handle_checks(results):
     def _open_db(self):
         self.db = adbapi.ConnectionPool('sqlite3', os.path.join(self.db_dir, "blockchainname.db"),
                                         check_same_thread=False)
-        return self.db.runQuery("create table if not exists name_metadata (" +
+
+        def create_tables(transaction):
```

Comment:
If we start adding more tables or doing more serious SQL work - we should look into sqlalchemy and alembic.  Otherwise we'll unnecessarily be reinventing the wheel for a lot of the basic db and data layer management.